imsv-docs-docusaurus/openapi/endpoints/prerequisites/spending-prerequisites.yaml

```diff
@@ -119,8 +119,8 @@ post:
         **FUNDING_SOURCE_INVALID**
         Funding source cannot be used with this card program.
 
-        **KYC_CARD_PROGRAM_REGION_MISMATCH**
-        The card program and the KYC profile are not is the same region.
+        **KYC_PROFILE_REGION_MISMATCH**
+        The KYC profile is already passed for a region different from the requested region. Use [Get a KYC profile](/get-a-kyc-profile/) endpoint to get the current region of the KYC profile. If the current region suits, request spending prerequisites again, while specifying that region, to get the remaining prerequisites. If your integration requires a specific region other than the current one, the user cannot proceed with getting their Immersve card and they must register with a new wallet.
```

Comment:
can we make this multiline so there is line break every 120 columns?
also, I ran this through grammar and spelling check and here is a new version:

> 
The KYC profile is already **approved** for a region different from the requested region. Use the [Get a KYC profile](/get-a-kyc-profile/) endpoint to determine the current region of the KYC profile.
If the current region is suitable, request spending prerequisites again, specifying that region, to obtain the remaining prerequisites. If your integration requires a specific region other than the current one, the user cannot proceed with obtaining their Immersive card and must register with a new wallet